src/main/webapp/assets/svg/github.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="15" height="16" viewBox="0 0 15 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I believe that svg elements can be given alt text/a title (see [here](https://css-tricks.com/accessible-svgs/#2-inline-svg) for example) for better accessibility! :)

Reply:
Sorry I didn't realize that the alt text goes on the `img` import of this file, my bad!

Reply:
No worries!